docker/Dockerfile.cuda-base

```diff
@@ -0,0 +1,18 @@
+# livepeer/cuda-base:11.7.1-cudnn8-devel-ubuntu20.04
+#
+# Base CUDA Develop image which contains CUDA SDK libs for the following architectures: linux amd64, linux arm64, windows amd64
+#
+# To build this image you need the following steps:
+# 1. Download NVIDIA CUDA SDK for ARM64, extract, and copy into cuda/arm64/usr/local/cuda/
```

Comment:
you can use the `ADD` directive in `Dockerfile`. https://stackoverflow.com/a/52069435/1190388

Reply:
Ohhh nice. I didn't know about it! I will leave it as it is right know and if at some point we decide to automate building cuda image, then I'll use this ADD + do cleanup of not needed files.